backend/resources/auth.py

```diff
@@ -66,6 +79,39 @@ def delete(self):
         return {"msg": "Successfully logged out"}, HTTPStatus.OK
 
 
+class RefreshAccessToken(Resource):
+    @jwt_refresh_token_required
+    def post(self):
+        jti = get_raw_jwt()['jti']
+        token = JWTToken.query.filter_by(jti=jti).one()
+
+        if token.revoked:
+            return {'msg': 'token has been revoked'}, HTTPStatus.UNAUTHORIZED
+
+        current_user = get_jwt_identity()
+        identity_claim = app.config['JWT_IDENTITY_CLAIM']
+
+        access_token = create_access_token(
+            identity=current_user, expires_delta=timedelta(minutes=60)
+        )
+        add_token_to_database(
+            access_token, identity_claim
+        )
+
+        ret = {'access_token': access_token}
+        return ret, HTTPStatus.CREATED
```

Comment:
I believe we could just use here: return {'access_token': access_token}, HTTPStatus.CREATED